### DISCUSSION TITLE
Error while running "Ludwig: Fine-Tune Llama-2-7b" in Colab

### DISCUSSION THREAD

**Discussion by leeywin** [ludwig-ai/ludwig#3851] [General]
Hi, recently I found "[Ludwig: Fine-Tune Llama-2-7b](https://colab.research.google.com/drive/1r4oSEwRJpYKBPM0M0RSh0pBEYK_gBKbe#scrollTo=nXOIxmfbQHSz)" in Colab could not run correctly.
It seems to have some version collision below.

`import getpass
import locale; locale.getpreferredencoding = lambda: "UTF-8"
import logging
import os
import torch
import yaml

from ludwig.api import LudwigModel

os.environ["HUGGING_FACE_HUB_TOKEN"] = getpass.getpass("Token:")
assert os.environ["HUGGING_FACE_HUB_TOKEN"]`

`---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
[<ipython-input-2-dac5961b998e>](https://localhost:8080/#) in <cell line: 5>()
      3 import logging
      4 import os
----> 5 import torch
      6 import yaml
      7 

12 frames
[/usr/lib/python3.10/_pyio.py](https://localhost:8080/#) in __init__(self, buffer, encoding, errors, newline, line_buffering, write_through)
   2043                 encoding = "utf-8"
   2044             else:
-> 2045                 encoding = locale.getpreferredencoding(False)
   2046 
   2047         if not isinstance(encoding, str):

TypeError: <lambda>() takes 0 positional arguments but 1 was given`

![截屏2024-01-02 20 36 13](https://github.com/ludwig-ai/ludwig/assets/29115621/49b53050-542a-4640-bcc1-dbdf45741fb4)

How can I make it work?

**Comment by arnavgarg1** [comment]
@leeywin Apologies for the terribly late response on this discussion item. Can you try this notebook version? https://colab.research.google.com/drive/1r4oSEwRJpYKBPM0M0RSh0pBEYK_gBKbe 

There was previously a strange error with locale setting in the notebook but this has been resolved since!
